src/scenes/MerchStorePage/Countdown.tsx

```diff
@@ -0,0 +1,112 @@
+import useTimer from 'hooks/useTimer';
+import { ALLOWLIST_MINTING_TIME, GALLERY_MINTING_TIME, PUBLIC_MINTING_TIME } from './times';
+import { useState, useEffect } from 'react';
+import { BaseS, BaseM, TitleM } from 'components/core/Text/Text';
+import styled from 'styled-components';
+import Spacer from 'components/core/Spacer/Spacer';
+import InteractiveLink from 'components/core/InteractiveLink/InteractiveLink';
+import colors from 'components/core/colors';
+
+export default function Countdown() {
+  const [nextTime, setNextTime] = useState('');
+  const [text, setText] = useState('');
+  const [showCounter, setShowCounter] = useState(true);
+  const [currentTime, setCurrentTime] = useState(new Date());
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCurrentTime(new Date());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, []);
+
+  useEffect(() => {
+    if (currentTime < new Date(ALLOWLIST_MINTING_TIME)) {
+      setNextTime(ALLOWLIST_MINTING_TIME);
+      setText('Shop opens for Premium Members in');
+    } else if (currentTime < new Date(GALLERY_MINTING_TIME)) {
+      setNextTime(GALLERY_MINTING_TIME);
+      setText('Shop opens for all Gallery Members in');
+    } else if (currentTime < new Date(PUBLIC_MINTING_TIME)) {
+      setNextTime(PUBLIC_MINTING_TIME);
+      setText('Shop opens to the general public in');
+    } else {
+      setShowCounter(false);
+    }
+  }, [currentTime]);
+
+  const { hours, minutes, seconds } = useTimer(nextTime);
+
+  return (
+    <>
+      {showCounter && (
+        <StyledContainer>
+          <BaseM>{text}</BaseM>
+          <Spacer height={12} />
+          <StyledCountdown>
+            <StyledCountdownText>
+              <StyledNumber>{hours === 'NaN' ? ' ' : hours}</StyledNumber>
```

Comment:
should be able to use [isNaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN)